stack.yaml

```diff
@@ -1,18 +1,14 @@
 # For more information, see: https://github.com/commercialhaskell/stack/blob/release/doc/yaml_configuration.md
 
 # Specifies the GHC version and set of packages available (e.g., lts-3.5, nightly-2015-09-21, ghc-7.10.2)
-resolver: lts-5.0
+resolver: nightly-2017-01-14
```

Comment:
The problem is that referencing a changing package set may lead to breakage even if there aren't any changes in the project. But let's try make it required for now, and if this job breaks too often, we can still revert it to an allowed failure.